python/interpret_community/mimic/mimic_explainer.py

```diff
@@ -313,6 +313,24 @@ def __init__(self, model, initialization_examples, explainable_model, explainabl
         self._original_eval_examples = None
         self._allow_all_transformations = allow_all_transformations
 
+    def _get_surrogate_model_predictions(self, evaluation_examples):
+        """Return the predictions given by the surrogate model."""
```

Comment:
add params and return to doc similar to other methods for sphinx doc

Reply:
Done